phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -1,130 +1,69 @@
-<!-- INCLUDE ucp_header.html -->
-
-<!-- IF not PROMPT -->
-	<!-- INCLUDE ucp_pm_message_header.html -->
-<!-- ENDIF -->
-
-<!-- IF PROMPT -->
-	<h2>{L_EXPORT_AS_CSV}</h2>
-	<form id="viewfolder" method="post" action="{S_PM_ACTION}">
-	<div class="panel">
-		<div class="inner">
-		<h3>{L_OPTIONS}</h3>
-		<fieldset>
-			<dl>
-				<dt><label for="delimiter">{L_DELIMITER}{L_COLON}</label></dt>
-				<dd><input class="inputbox" type="text" id="delimiter" name="delimiter" value="," /></dd>
-			</dl>
-			<dl>
-				<dt><label for="enclosure">{L_ENCLOSURE}{L_COLON}</label></dt>
-				<dd><input class="inputbox" type="text" id="enclosure" name="enclosure" value="&#034;" /></dd>
-			</dl>
-		</fieldset>
+{% include 'ucp_header.html' %}
+
+{% if not PROMPT %}
+{% include 'ucp_pm_message_header.html' %}
+{% endif %}
+
+{% if messagerow %}
+<div class="pm">
+	<ul class="pm-list">
+		{% for message in messagerow %}
+		<li class="pm-item">
+			<span class="pm-primary">
+				{% if message.S_PM_UNREAD %}
+					<i class="fa fa-certificate fa-fw pm-unread" title="{{ lang('UCP_PM_UNREAD') }}" aria-label="{{ lang('UCP_PM_UNREAD') }}"></i>
+				{% endif %}
+
+				{% if message.S_PM_MARKED %}
+					<i class="fa fa-star fa-fw pm-marked" title="{{ lang('UCP_PM_MARKED') }}" aria-label="{{ lang('UCP_PM_MARKED') }}"></i>
+				{% endif %}
+
+				<a href="{{ message.U_VIEW_PM }}" title="{{ message.SUBJECT }}" class="pm-title {{ message.FOLDER_IMG_STYLE == 'pm_unread' ? 'pm-unread' }}">{{ message.SUBJECT }}</a>
+				<span class="pm-secondary">
+					<span class="pm-secondary-item pm-authour">
+						{% if message.PM_CLASS == "pm_foe_colour" %}
```

Comment:
This must be resolved better. As I have shown you before. The solution is almost working, you just need to find why it wasn't assigned correctly. Add new variable named `S_AUTHOR_FOE` and use it instead of this condition.